### DISCUSSION TITLE
Laser emmiter 45

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4138] [Feature Suggestions]
![2022-05-03_16-51-56](https://user-images.githubusercontent.com/104623098/166448019-1baf28ec-e21c-4dc0-97e9-be4e26afd3b1.png)
![sp_a2_triple_laser0000](https://user-images.githubusercontent.com/104623098/166448034-0f8b291d-a1d7-4044-9274-4d0a886d8282.jpg)
You just need to add a laser emitter to the 45 degree block, and the "cube type:" will be responsible for the position of the laser in the block (Side, middle and top with bottom (Only 1 side is possible, the block can also be rotated as an incline surface)

**Comment by ItzAlmaZ** [comment]
⠀⠀⠀⠀⣀⣤⣤⣤⣤⣀⠀⠀⠀⠀
⠀⠀⣴⣿⣿⣿⣿⣿⣿⣿⣿⣦⠀⠀
⠀⣼⣿⣿⠉⠉⠉⠉⡏⠉⣿⣿⣧⠀
⠀⣿⣿⡿⠀⠀⠀⠀⡇⠀⣿⣿⣿⠀
⠀⢻⣿⣧⣤⣤⡄⠀⣧⣤⣿⣿⡟⠀
⠀⠀⠻⣿⣿⣿⣿⣾⣿⣿⣿⠟
⠀⠀⠀⠀⠉⠛⠛⠛⠛⠉⠀⠀⠀⠀⠀

**Comment by SimpleFoxOfficial** [comment]
like it. But i think this creating with hammer
